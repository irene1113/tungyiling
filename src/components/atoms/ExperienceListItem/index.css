.experience-list-item {
	position: relative;
	padding-left: 30px;
	color: var(--black);

	&::before {
		display: inline-flex;
		content: '';
		width: 6px;
		height: 6px;
		position: absolute;
		left: 0;
		top: 7px;
		background: var(--gray);
	}

	& > .duration {
		font-size: 18px;
		font-family: 'Puritan', sans-serif;
		margin-bottom: 10px;
	}

	& > .content {
		font-size: 14px;
		font-weight: 300;
	}
}
