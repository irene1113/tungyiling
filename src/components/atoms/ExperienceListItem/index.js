import React from 'react';
import classnames from 'classnames';

import styles from './index.css';

const ExperienceListItem = ({ className, duration = '', content = '' }) => (
	<div className={classnames(styles.experienceListItem, className)}>
		<div className={styles.duration}>{duration}</div>
		<div className={styles.content}>{content}</div>
	</div>
);

export default ExperienceListItem;
