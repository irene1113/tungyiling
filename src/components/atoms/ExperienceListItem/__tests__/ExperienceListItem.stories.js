import React from 'react';
import { storiesOf } from '@storybook/react';
import { text } from '@storybook/addon-knobs/react';

import ExperienceListItem from 'components/atoms/ExperienceListItem';

const stories = storiesOf('atoms/ExperienceListItem', module);

stories.add('__interactive', () => (
	<ExperienceListItem
		duration={text('duration', '2017 - Now')}
		content={text('content', '25sprout, Frontend Developer')}
	/>
));
