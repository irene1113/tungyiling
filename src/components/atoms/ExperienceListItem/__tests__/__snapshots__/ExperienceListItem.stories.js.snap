// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="experienceListItem"
  >
    <div
      class="duration"
    >
      2017 - Now
    </div>
    <div
      class="content"
    >
      25sprout, Frontend Developer
    </div>
  </div>
</DocumentFragment>
`;
